src/karapace/backup/backends/v3/schema_tool.py

```diff
@@ -58,6 +58,17 @@ def relative_path(path: pathlib.Path) -> pathlib.Path:
     return pathlib.Path(str_path[len(cwd) + 1 :]) if str_path.startswith(cwd) else path
 
 
+def target_has_source_layout(git_target: str) -> bool:
```

Comment:
thought (non-blocking): This is OK, but I think would have allowed this check to fail for this PR instead. The trade-off is that the PR is now not purely a clean move into `src/`.